docs/HowToFAQ/FAQTerminationFalse.md

```diff
@@ -0,0 +1,47 @@
+---
+title: What does {:termination false} do on trait? It looks like it is required if I want to extend traits from other modules.
```

Comment:
```suggestion
title: What does `{:termination false}` do on trait? It looks like it is required if I want to extend traits from other modules.
```

Reply:
Markdown does not apply that kind of formatting in the title, unfortunately.